### DISCUSSION TITLE
presigned url logging

### DISCUSSION THREAD

**Discussion by rstanonik** [minio/minio#17542] [Q&A] [answered]
We have audit and logger enabled, but don't see anything when a presigned url is created, "aws s3 presigned"
We do see logging when the presigned url is used to download and for actions such as "aws s3 ls", "aws s3 cp", etc.
Thanks,
Ron

**Comment by harshavardhana** [comment, marked as answer]
Presigned URL is a purely client side operation with no interaction over the network.

**Reply by rstanonik** [reply]
Thanks!
